examples/https/README.adoc

```diff
@@ -0,0 +1,234 @@
+
+It is a simple MicroProfile application that uses JAX-RS to expose HTTP endpoints that displays text configured with MicroProfile Config.
+
+This Helm Chart will show how to deploy it on OpenShift with HTTPS configured.
+
+## Prerequisites
+
+In order to use HTTPS with this application, you need to create a keystore that contains the self-signed certificate and put it in a secret in OpenShift.
+
+### Generate a keystore
+
+You can use the Java `keytool` command to generate a keystore:
+
+[source,bash]
+----
+keytool -genkey -keyalg RSA -alias <alias_name> -keystore <keystore_filename.jks> -validity 360 -keysize 2048
+----
+
+For example, for this application , use the following command to generate a keystore:
+
+[source,bash]
+----
+keytool -genkey -keyalg RSA -alias eapdemo-selfsigned -keystore keystore.jks -validity 360 -keysize 2048
+----
+
+Note the `keystore password` that you are typing when prompted as we will need it later.
+
+### Create a secret for this keystore
+
+In order to use that keystore on OpenShift, we need to store it in a Secret. We will also store the `keystore password` in that secret in order to read from the keystore.
+
+Create a secret from the previously created keystore and its secret using the following command:
+
+[source,bash]
+----
+oc create secret generic <secret_name> \    
+  --from-file=<keystore_filename.jks> \
+  --from-literal=keystore-password=<the keystore password>
+----
+
+For example, for this application, use the following command to create a secret:
+
+[source,bash]
+----
+oc create secret generic wildfly-app-secret \
+  --from-file=keystore.jks \
+  --from-literal=keystore-password=mypasswordxyz
+----
+
+[NOTE]
+====
+I passed `mypasswordxyz` as the value of the keystore password as it is the password I typed when prompted by the `keytool` command. You will have to adjust that value to put your own password. 
+====
+
+## Installation
+
+To deploy the application on OpenShift, we use the `helm` tool.
+
+[source,bash]
+----
+helm install wildfly-app -f examples/https/helm.yaml ./charts/wildfly
+----
+
+The `helm.yaml` configures WildFly to enable HTTPS.
+The relevant part are:
+
+[source,yaml]
+----
+build:
+  s2i:
+    featurePacks:
+      - org.wildfly.cloud:wildfly-cloud-galleon-pack:1.0.1.Final
+----
+
+When we build the application image, we provision the 
+`org.wildfly.cloud:wildfly-cloud-galleon-pack` Galleon Pack that will adapt WildFly on the Cloud.
+In particular, it will allow to use a few environment variables to automatically configure HTTPS.
+The list of environment variables supported by the WildFly CLoud Galleon Pack is listed in its https://github.com/wildfly-extras/wildfly-cloud-galleon-pack/blob/main/doc/index.md[documentation].
+
+Let's specify these HTTPS environment variables when the application image is deployed:
+
+[source,yaml]
+----
+deploy:
+  env:
+    - name: HTTPS_KEYSTORE
+      value: keystore.jks
+    - name: HTTPS_KEYSTORE_TYPE
```

Comment:
This env is not rquired, elytron will discover that the keystore is JKS

Reply:
thanks, it works without it!

@jfdenise However this is not how I understood its docs at https://github.com/wildfly/wildfly-cekit-modules/blob/5a2927a952d07fd996fabd7f99d1b56d20053947/jboss/container/wildfly/launch/elytron/module.yaml#L31.

> The underlying type of the keystore (JKS, JCEKS, etc.). Defaults to JCEKS

The doc should be updated to specify that if not defined, Elytron will automatically discover the the type of the keystore.